model/build/build.go

```diff
@@ -172,6 +173,17 @@ func (b *Build) UpdateStatus(status string) error {
 	)
 }
 
+func (b *Build) SetAbortedStatus(aborted bool) error {
```

Comment:
This should just say SetAborted; "Status" is confusing since this doesn't actually set status

Reply:
+1

Reply:
done